mangaki/mangaki/templates/mangaki/work_detail.html

```diff
@@ -139,6 +139,10 @@
 	(cc) <a href="http://manga-news.com/" target="_blank">Manga-News</a>
       {% endif %}
     </div>
+{% elif object.ext_synopsis %}
```

Comment:
Honestly, I prefer English synopsis over no synopsis at all.
But could be a profile-specific preference, what's your take?